src/llmcompressor/transformers/compression/quantization_format.py

```diff
@@ -16,7 +16,7 @@ def infer_quantization_format(
     model,
     quantization_format: Optional[str] = None,
     save_compressed: bool = False,
-    sparsity_config: Optional[SparsityCompressionConfig] = None,
+    sparsity_structure: Optional[str] = None,
```

Comment:
Let's add the table to the docstring once completed

Reply:
Added!